build/templates/_converters.py.mako

```diff
@@ -247,6 +248,26 @@ def convert_double_each_element(numbers):
     return [x * 2 for x in numbers]
 
 
+% endif
+<%
+# There are some parameters in nidigital that are a single site so not strictly a repeated capability
```

Comment:
I think a more accurate reason for this converter is - There are some parameters in nidigital that cannot be made into a repeated capability because the methods already have a repeated capability (`pins`), and we want the parameter to behave similarly to repeated capabilities.